relay-event-schema/src/protocol/span.rs

```diff
@@ -179,6 +179,150 @@ impl Getter for Span {
     }
 }
 
+/// Indexable fields added by sentry (server-side).
+// TODO: types
```

Comment:
I considered typing fields in this struct as booleans or integers where appropriate, but this would make (de)serialization harder, because at the end we want string values in the kafka consumer.

```suggestion
```